packages/js/product-editor/src/blocks/generic/section-description/edit.tsx

```diff
@@ -2,28 +2,21 @@
  * External dependencies
  */
 import { createElement } from '@wordpress/element';
-import { useWooBlockProps } from '@woocommerce/block-templates';
 
 /**
  * Internal dependencies
  */
-import { BlockFill } from '../../../components/block-slot-fill';
 import { ProductEditorBlockEditProps } from '../../../types';
 import { SectionDescriptionBlockAttributes } from './types';
 
 export function SectionDescriptionBlockEdit( {
 	attributes,
 }: ProductEditorBlockEditProps< SectionDescriptionBlockAttributes > ) {
 	const { content } = attributes;
-	const blockProps = useWooBlockProps( attributes );
 
 	return (
-		<BlockFill
-			{ ...blockProps }
-			name="section-description"
-			slotContainerBlockName="woocommerce/product-section"
-		>
+		<div>
```

Comment:
I'm wondering how we are going to deal with this now that is gone.
This allow us to render a specific child as part of the section header.

Reply:
Ya, I'm not entirely sure yet.  Is this mostly used for the "Change product type" dropdown?  Any other use cases?

Off the top of my head, I see a couple options:

1. Use a classic `SlotFill` approach (this may have already been attempted)
2. Allow block insertion in these areas.  This may be a bit easier with this new "view" concept since we're dealing with normal React props